omod/src/main/webapp/resources/scripts/directives/select-drug.js

```diff
@@ -36,7 +36,7 @@ angular.module('uicommons.widget.select-drug', [ 'drugService', 'ui.bootstrap' ]
                 }
             },
             template: '<input type="text" id="{{ inputId }}" ng-model="ngModel" ng-blur="verify()" ' +
-                'typeahead="drug as drug.display for drug in search($viewValue) | filter:$viewValue" ' +
+                'typeahead="drug as drug.display for drug in search($viewValue) " ' +
                 'typeahead-on-select="onSelect($item, $model, $label)" ' +
                 'typeahead-editable="false" autocomplete="off" placeholder="{{ placeholder }}" autocomplete="off" ' +
                 'ng-required="{{ required }}" size="{{ size }}" ' +
```

Comment:
From [this](https://talk.openmrs.org/t/options-for-openmrs-drug-management/1719/32) talk topic, seems like there is more to be done.

Reply:
There isn't more on this module. The other piece of work is supposed to be done on the order entry ui module.

Reply:
@wluyima even if this code works it is redundant. The results sent from the backend are already filtered based on the typed in characters, why filter the results again?

Reply:
Makes sense